observation/src/main/java/com/linecorp/armeria/server/observation/MicrometerObservationService.java

```diff
@@ -82,7 +69,7 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
 
         final HttpServerContext httpServerContext = new HttpServerContext(ctx, req);
         final Observation observation = ServiceObservationDocumentation.OBSERVATION.observation(
-                this.serviceObservationConvention, DefaultServiceObservationConvention.INSTANCE,
+                null, DefaultServiceObservationConvention.INSTANCE,
```

Comment:
So I understand that you're not allowing users to modify the default tagging by injecting the custom convention?